gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt

```diff
@@ -37,11 +38,59 @@ class ComposePlugin : Plugin<Project> {
             }
 
             project.dependencies.add(
-                    "kotlinCompilerPluginClasspath",
-                    "org.jetbrains.compose.compiler:compiler:$composeVersion"
+                "kotlinCompilerPluginClasspath",
+                "org.jetbrains.compose.compiler:compiler:$composeVersion"
             )
         }
 
+        fun ComponentModuleMetadataHandler.replaceAndroidx(original: String, replacement: String) {
```

Comment:
Should the replacements be configurable? (i.e. should there be a flag to disable replacements?)

Reply:
I thought about it. But haven't figure out where it might be useful.

In all cases when we have two libraries in classpath ("androidx.compose" and "org.jetbrains.compose") we always will have an error.

If we have only one library, then everything will work as before.